core/src/env/srml/srml_only/impls.rs

```diff
@@ -72,6 +72,18 @@ impl EnvStorage for SrmlEnvStorage {
     }
 }
 
+/// Wrapped access to return data.
+/// get() Must be used right after ReturnData is returned.
+#[allow(unused)]
+pub struct ReturnData {}
+
+#[allow(unused)]
```

Comment:
why?

Reply:
Because sandbox(ext_sandbox_size and ext_sandbox_copy) is overwritten when another message is called.
So if call() is used several times in one contract method, we should use ReturnData::get() before next call() is used.

Reply:
Yeah this is a problem.
So we have to make sure that this situation does not happen.
Normally users of an API are forced to use whatever they got using the `#[must_use]` attribute.